bears/markdown/MarkdownBear.py

```diff
@@ -136,13 +141,16 @@ def create_arguments(filename, file, config_file,
         # Remove { and } as remark adds them on its own
         settings = config_json[1:-1]
 
-        args = ['--no-color', '--quiet', '--setting', settings]
+        args = [filename, '--no-color', '--quiet', '--setting', settings]
```

Comment:
Yes, `use_stdin` doesn't work with `validate-links` :smile: I've removed the `use_stdin` though :+1: